index.html

```diff
@@ -283,13 +285,83 @@ <h2>Documentation</h2>
 
         <div class="container">
 
-        <!-- mainline releases v4.11.0.3 -->
+        <!-- mainline releases v4.11.0.4 -->
 
         <!--<div class="break"></div>insert this code when a new update comes out to separate a new version from the current.-->
 
-        <div class="smaller-margin" id="mr-31">
+        <div class="smaller-margin" id="mr-32">
             <h2 class="text-middle">Mainline releases</h2>
             <!-- h2 tag needs to be moved to the newest version -->
+            <h4><a target="_blank" href="https://github.com/pcengines/coreboot/compare/v4.11.0.3...v4.11.0.4">v4.11.0.4</a></h4>
+        </div>
+
+        <div class="dimmed-text">
+            <p>Release date: '2020-02-26'</p>
```

Comment:
@piotr-kleins check all the variables that gen-index.py uses. Many of the may have the apostrophes included. Please remove them wherever appropriate

Reply:
Fixed